packages/rpc/test/marshal.test.ts

```diff
@@ -0,0 +1,11 @@
+import '@endo/init';
+import { expect, test } from 'vitest';
+import { makeClientMarshaller } from '../src/marshal.js';
+
+test('inclue boardIDs in string version of synthetic remotables', () => {
+  const m = makeClientMarshaller();
+  const capData = { body: '#"$0.Brand"', slots: ['board0123'] };
+  const x = m.fromCapData(capData);
+  const xstr = `${x}`;
+  expect(xstr).toBe('[object Alleged: Brand#board0123]');
```

Comment:
I don't know if we have style on this but I would argue for an explicit type conversion, which works the same for all primitive types.
```suggestion
  const str = String(m.fromCapData(capData));
  expect(str).toBe('[object Alleged: Brand#board0123]');
```

Reply:
template string is the coercion idiom I've seen most often, though early on I did wonder what it was doing there.

I wasn't clear on whether `String(x)` actually does the same thing, but some double-checking confirms that it does:

> When `String` is called as a constructor (with new), it creates a [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) object, which is not a primitive.
> 
> When `String` is called as a function, it coerces the parameter to a string primitive.
-- [String\(\) constructor \- JavaScript \| MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/String#return_value)